code/datums/components/fantasy/prefixes.dm

```diff
@@ -137,7 +137,7 @@
 		/datum/reagent/toxin/histamine = "creeping malaise"
 	)
 	var/poisonname = names[picked_poison]
-	master.AddElement(/datum/element/venomous, picked_poison, comp.quality+1)
+	master.AddElement(/datum/element/venomous, picked_poison, comp.quality+1, INJECT)
```

Comment:
You should be specifying the argument for this, see: https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#prefer-named-arguments-when-the-meaning-is-not-obvious